prisma/migrations/20230716161726_initial/migration.sql

```diff
@@ -0,0 +1,1045 @@
+-- MANUAL CHANGES:
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
For some reason we use a Postgres Plugin to generate UUIDs on the database side, which for sure Prisma did not know. Thus `prisma db push` also does not work, but with the changes here `prisma migrate reset` works